src/db/mongodb/collection.rs

```diff
@@ -65,6 +68,28 @@ pub trait MongoDbCollectionExt: MongoDbCollection {
         self.collection().create_index(index, options).await
     }
 
+    /// Calls [`mongodb::Collection::drop_index()`] and coerces the document type.
+    /// Also, ignores already missing indexes.
+    async fn drop_index(
+        &self,
+        name: impl AsRef<str> + Send + Sync,
+        options: impl Into<Option<DropIndexOptions>> + Send + Sync,
+    ) -> Result<(), Error> {
+        match self.collection().drop_index(name, options).await {
+            Err(e) => match &*e.kind {
+                ErrorKind::Command(c) => {
+                    if c.code == INDEX_NOT_FOUND_CODE {
+                        Ok(())
```

Comment:
really? :see_no_evil:

Reply:
Thinking about this we don't need that code at all. The migration should fail if the indexes are not their, we could see it as a safeguard.

Reply:
If we ever want to ignore these errors it should be in the migration specific code.

Reply:
alrighty ... let's resolve and never look at it again then :trollface: